.github/workflows/regression-test-PROD.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       uses: grafana/k6-action@v0.3.1
       with:
         filename: test/k6/src/tests/instances.js
-        flags: -e runFullTestSet=true -e env=${{ vars.ENV }} -e org=${{ vars.ORG }} -e app=${{ vars.APP }} -e userName=${{ secrets.REGRESSION_USER_NAME}} -e userPassword=${{ secrets.REGRESSION_USER_PASSWORD}} -e apimSubsKey=${{ secrets.APIM_SUBSKEY }} -e mpClientId=${{ secrets.MP_CLIENT_ID }} -e mpKid=${{ secrets.MP_KID }} -e encodedJwk=${{ secrets.MP_ENCODEDJWK }}
+        flags: -e runFullTestSet=true -e env=${{ vars.ENV }} -e pid=${{ secrets.PERSON_NUMBER }} -e partyId=${{ secrets.PARTY_ID }} -e orgNumber=${{ secrets.ORG_NUMBER }} -e org=${{ vars.ORG }} -e app=${{ vars.APP }} -e userName=${{ secrets.REGRESSION_USER_NAME}} -e userPassword=${{ secrets.REGRESSION_USER_PASSWORD}} -e apimSubsKey=${{ secrets.APIM_SUBSKEY }} -e mpClientId=${{ secrets.MP_CLIENT_ID }} -e mpKid=${{ secrets.MP_KID }} -e encodedJwk=${{ secrets.MP_ENCODEDJWK }}
```

Comment:
why is partyId required here?

Reply:
Yea! Correct! We don't need party ID when using basic auth! 👍

Reply:
so the property should be removed. Unresolving

Reply:
My bad! Double checked this time! `-e partyId=${{ secrets.PARTY_ID }}` removed from all the 4 files. 👍